### PR TITLE
AESinkAudiotrack: Properly use periods for buffers

### PR DESCRIPTION
**Pull request by fritsch**
WIP: Try to find a sensible ratio for RAW PT as well.

I cannot remember anymore why I implemented it the other way -> 5? years ago :-)

Maybe I will remember

### CODE REVIEW

**Comment by thexai** [comment]
Tested (RAW only, as is only what changes):

**AC3**
Play: 100% OK
Pause: 100% OK
Seek/Chapter Skip: 100 OK

**DD+ Atmos**
Play: 100% OK
Pause: 100% OK
Seek/Chapter Skip: 100% OK

**TrueHD Atmos**
Play: 95% OK (in one movie only the sound takes time to synchronize at the beginning (or goes away momentarily) and there is an initial sound that is not fully heard. Before this PR and maybe https://github.com/xbmc/xbmc/pull/22392 I'm sure this not happen on same movie).
Pause: 60% OK
Seek/Chapter Skip: 70% OK

**DTS**
Play: 100% OK
Pause: 100% OK
Seek/Chapter Skip: 100% OK

**DTS-HD MA**
Play: 100% OK
Pause: 100% OK
Seek/Chapter Skip: 50% OK

In summary: improves **DTS-HD MA Pause**, the rest does not seem to change much except a possible regression in TrueHD. It seems that it is not ready yet and more research is needed.

Today no time to do more test or collect logs.

**Comment by thexai** [comment]
Then introduced in https://github.com/xbmc/xbmc/pull/22392 but anyway is an improvement as before Pause 0%, Seek 0%

**Comment by thexai** [comment]
Tested DTS-HD MA with update and no differences found: pause still working 100% and chapter skip 50%

**Comment by fritsch** [comment]
@thexai without this PR both of the working things is broken right?

**Comment by thexai** [comment]
Think this PR "only" improves DTS-HD MA Pause and no regressions in the rest.

DTS-HD MA Seek/Chapter skip was already improved in https://github.com/xbmc/xbmc/pull/22392

**Comment by fritsch** [comment]
I am closing this one - I have some more rework on the way for RAW.
